src/components/Reportees/index.jsx

```diff
@@ -6,58 +6,159 @@ import TableBody from '@material-ui/core/TableBody';
-          ))}
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {Object.values(partialOrg)
-          .filter(({ cn }) => cn !== ldapEmail)
-          .sort(sortByPersonName)
-          .map(({ cn, mail, bugzillaEmail }) => (
-            <TableRow key={mail}>
-              <TableCell key={mail}>{`${cn} `}</TableCell>
-              {Object.keys(CONFIG.reporteesMetrics).map((metricUid) => {
-                const countLink = ((metrics || {})[bugzillaEmail] || {})[metricUid];
-                return (
-                  <TableCell align="right" key={metricUid}>
-                    {countLink && (
-                      <a
-                        key={countLink.link}
-                        href={countLink.link}
-                        target="_blank"
-                        rel="noopener noreferrer"
-                      >
-                        {countLink.count}
-                      </a>
-                    )}
-                  </TableCell>
-                );
-              })}
-            </TableRow>
-          ))}
-      </TableBody>
-    </Table>
-  </div>
-);
 
+function desc(a, b, orderBy) {
+  const valueA = a[orderBy];
+  const valueB = b[orderBy];
+
+  const countA = valueA ? valueA.count : 0;
+  const countB = valueB ? valueB.count : 0;
+
+  if (countB < countA) {
+    return -1;
+  }
+
+  if (countB > countA) {
+    return 1;
+  }
+
+  return 0;
+}
+
+function stableSort(array, cmp) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = cmp(a[0], b[0]);
+    if (order !== 0) return order;
+    return a[1] - b[1];
+  });
+  const result = stabilizedThis.map(el => el[0]);
+  return result;
+}
+
+function getSorting(order, orderBy) {
+  return order === 'desc' ? (a, b) => desc(a, b, orderBy) : (a, b) => -desc(a, b, orderBy);
+}
+
+
+class Reportees extends React.PureComponent {
+  state = {
+    orderBy: 'cn',
+    order: 'asc',
+  };
+
+  getMergedProps() {
+    const { metrics, partialOrg, ldapEmail } = this.props;
+
+    // filter out the manager
+    const reportees = Object.values(partialOrg)
+      .filter(({ mail }) => mail !== ldapEmail);
+
+    // add metrics
+    const reporteesWithMetrics = reportees.map(reportee => ({
+      ...reportee,
+      ...metrics[reportee.bugzillaEmail],
+    }));
+
+    return reporteesWithMetrics;
+  }
+
+  createSortHandler = property => (event) => {
+    this.handleRequestSort(event, property);
+  };
+
+  handleRequestSort = (event, property) => {
+    const { order, orderBy } = this.state;
+
+    let newOrder = 'desc';
+    if (orderBy === property && order === 'desc') {
+      newOrder = 'asc';
+    }
+
+    this.setState({
+      orderBy: property,
+      order: newOrder,
+    });
+  };
+
+  render() {
+    const { classes } = this.props;
+
+    const { order, orderBy } = this.state;
+
+    return (
+      <div className={classes.root}>
+        <Table>
+          <TableHead>
```

Comment:
Instead of

![Screen Shot 2019-05-03 at 9 17 27 AM](https://user-images.githubusercontent.com/3766511/57139861-4f984280-6d84-11e9-835e-d0ecf7b55d1b.png)

We could probably put everything in a single line:

![Screen Shot 2019-05-03 at 9 16 14 AM](https://user-images.githubusercontent.com/3766511/57139879-59ba4100-6d84-11e9-9e48-a3d0f00651a7.png)